packages/nimble-components/src/rich-text/editor/models/create-tiptap-editor.ts

```diff
@@ -15,27 +15,31 @@ import HardBreak from '@tiptap/extension-hard-break';
 import { Slice, Fragment, Node as FragmentNode } from 'prosemirror-model';
 import { PluginKey } from 'prosemirror-state';
 
+import { keyEscape } from '@microsoft/fast-web-utilities';
 import { mentionPluginPrefix } from '../types';
 
 import { anchorTag } from '../../../anchor';
 import type { MentionExtensionConfiguration } from '../../models/mention-extension-configuration';
+import type { RichTextMentionListBox } from '../../mention-list-box';
+import type { RichTextEditor } from '..';
 
 const validAbsoluteLinkRegex = /^https?:\/\//i;
 
 export function createTiptapEditor(
-    editor: HTMLDivElement,
-    mentionExtensionConfig: MentionExtensionConfiguration[]
+    richTextEditor: RichTextEditor,
```

Comment:
Think we should avoid coupling this to the entire editor element if possible

Reply:
We could do something similar to `mentionUpdateEmitter` i.e. pass in a function like `activeMentionCharacterEmitter` that is passed the active character.